code/game/machinery/computer/camera_advanced.dm

```diff
@@ -74,17 +74,18 @@
 	CRASH("[type] does not implement ai eye handling")
 
 /obj/machinery/computer/camera_advanced/remove_eye_control(mob/living/user)
-	if(!user)
+	if(isnull(user?.client))
 		return
+
 	for(var/datum/action/actions_removed as anything in actions)
 		actions_removed.Remove(user)
 	for(var/datum/camerachunk/camerachunks_gone as anything in eyeobj.visibleCameraChunks)
 		camerachunks_gone.remove(eyeobj)
-	if(user.client)
-		user.reset_perspective(null)
-		if(eyeobj.visible_icon && user.client)
-			user.client.images -= eyeobj.user_image
-		user.client.view_size.unsupress()
+
+	user.reset_perspective(null)
+	if(eyeobj.visible_icon && user.client)
```

Comment:
```suggestion
	if(eyeobj.visible_icon)
```

Cause we already check for null client in the beginning of this proc so its redundant